components/eam/docs/tech-guide/orodrag.md

```diff
@@ -0,0 +1,35 @@
+# Orographic drag scheme
+
+## Overview
+
+The orographic drag schemes includes two main options: the default Gravity Wave Drag schemeof McFarlane (1987)[@mcfarlane_the_1987] and a new suite of orographic drag parameterization schemes. The new suite includes 4 components all combined in one module (i.e. subroutine gwdo2d). The schemes include orographic gravity wave drag (oGWD, Xie et al.,2020)[@xie_an_2020], flow-blocking drag (FBD, Xie et al.,2020)[@xie_an_2020], small-scale GWD (sGWD, Tsiringakis et al.,2017)[@tsiringakis_small_2020], and turbulent-scale orographic form drag (TOFD, Beljaars et al., 2004)[@beljaars_a_2020]. The oGWD and TOFD schemes are used to replace the default oGWD (McFarlane, 1987)[@mcfarlane_the_1987] and Turbulent Mountain Stress (TMS, documented on Richter et al., 2010)[@richter_the_2010] module in EAM, while the FBD and sGWD are added enhanced drag schemes. Each of the schemes are used to predict and add enhanced drags from surface to high level that would help decelerate the wind especially over the mountainous regions among the globe. The oGWD, FBD, and sGWD are implemented in gw_drag.F90, while TOFD is implemented in clubb_intr.F90 to join the vertical diffusion process. There are also new topographic parameters (orographic asymmetry [OA], orographic convexity [OC], effective orographic length [OL]) are input into the model for the new oGWD and FBD scheme implmented. The conceptual location of each scheme is illustrate below.
+
+![orodrag figure](../figures/orodrag.png)
+
+### Default oGWD scheme
+
+The current default oGWD scheme in E3SMv3 is from McFarlane (1987)[@mcfarlane_the_1987]. It is a linear orographic gravity wave drag scheme that parameterizes the subgrid process of vertical propagation of gravity wave originated from orographic source. The output from this scheme is a vertical profile of drag (or deceleration terms) when gravity wave breaks at higher levels and deposit momemtum flux to that level. This scheme is shown to have impact on excessive westerly wind in the extratropics and the wind bias in the polar region.
```

Comment:
Note that it is turned on in E3SMv3.0. 

Grammatical fixes: 
Replace `originated from` with `originating from`
Replace `and deposit momentum` with `and deposits momentum` 
Replace `have impact on` with `impact` (or better yet replace with `improve`, if that is true)

Reply:
You want something more then "The current default oGWD scheme in E3SMv3 is from McFarlane " ?

Reply:
I think Chris just wants to note that this scheme is turned on by default in E3SMv3.0

Reply:
Jinbo's recent revisions have addressed my comment about just noting what is turned on and off by default in E3SMv3.0.